packages/db/src/controller/level.ts

```diff
@@ -20,6 +20,9 @@ export type LevelDbControllerModules = {
 
 const BUCKET_ID_UNKNOWN = "unknown";
 
+/** Time between capturing metric for db size, every few minutes is sufficient */
+const DB_SIZE_METRIC_INTERVAL_MS = 5 * 60 * 1000;
```

Comment:
every 5 minutes seems to be sufficient as we don't really need real-time data for this, it is just an approximation after all

Reply:
`approximateSize` takes around 0.05 - 0.5ms and it is executed in a different thread, so we could definitely also increase this interval if necessary